classifiers/python_functions/textblob_subjectivity/code_snippet.md

```diff
@@ -1,16 +1,17 @@
 ```python
 from textblob import TextBlob
+from typing import Dict
 
-YOUR_ATTRIBUTE = "text"
+YOUR_ATTRIBUTE: str = "text"
 
 def setall(d, keys, value):
     for k in keys:
         d[k] = value
 
-MAX_SCORE = 100
-MIN_SCORE = 0
+MAX_SCORE: int = 100
```

Comment:
see above

Reply:
changed to working version for refinery https://github.com/code-kern-ai/bricks/blob/17cb2b4bcdfad8c4fc3e215d844305771372a4af/classifiers/python_functions/textblob_subjectivity/code_snippet.md